### ISSUE TITLE
next release date ?

### ISSUE THREAD

**Issue by gitfy** [bytedeco/javacpp-presets#498]
Hi,
Is there a tentative date when the next release will happen. The reason to check, there was a critical bug which fftw 3.3.6 addresses and it would be good to have that preset binary build ready to download.
Thanks

**Comment by saudet** [comment]
SNAPSHOT artifacts are available, so you could use those.

**Comment by gitfy** [comment]
I would like to point to the snapshot, but wasn't successful in identifying the repo & the corresponding gradle dependency setting. If you could help get the right settings, it would be helpful.
Thanks for your response.

**Comment by saudet** [comment]
Check the information here: http://bytedeco.org/builds/

**Comment by gitfy** [comment]
Thanks

**Comment by gitfy** [comment]
I tried it and couldn't get the following

The version which is mentioned in the readme:
compile group: 'org.bytedeco.javacpp-presets', name: 'fftw-platform', version: '3.3.7-1.3.4-SNAPSHOT'

```
Could not resolve all files for configuration ':compileClasspath'.
> Could not find fftw-android-arm.jar (org.bytedeco.javacpp-presets:fftw:3.3.7-1.3.4-SNAPSHOT).
  Searched in the following locations:
      https://oss.sonatype.org/content/repositories/snapshots/org/bytedeco/javacpp-presets/fftw/3.3.7-1.3.4-SNAPSHOT/fftw-3.3.7-1.3.4-20171215.002004-377-android-arm.jar
> Could not find fftw-android-x86.jar (org.bytedeco.javacpp-presets:fftw:3.3.7-1.3.4-SNAPSHOT).
  Searched in the following locations:
      https://oss.sonatype.org/content/repositories/snapshots/org/bytedeco/javacpp-presets/fftw/3.3.7-1.3.4-SNAPSHOT/fftw-3.3.7-1.3.4-20171215.002004-377-android-x86.jar
> Could not find fftw-linux-x86.jar (org.bytedeco.javacpp-presets:fftw:3.3.7-1.3.4-SNAPSHOT).
  Searched in the following locations:
      https://oss.sonatype.org/content/repositories/snapshots/org/bytedeco/javacpp-presets/fftw/3.3.7-1.3.4-SNAPSHOT/fftw-3.3.7-1.3.4-20171215.002004-377-linux-x86.jar
> Could not find fftw-linux-x86_64.jar (org.bytedeco.javacpp-presets:fftw:3.3.7-1.3.4-SNAPSHOT).
  Searched in the following locations:
      https://oss.sonatype.org/content/repositories/snapshots/org/bytedeco/javacpp-presets/fftw/3.3.7-1.3.4-SNAPSHOT/fftw-3.3.7-1.3.4-20171215.002004-377-linux-x86_64.jar
> Could not find fftw-linux-armhf.jar (org.bytedeco.javacpp-presets:fftw:3.3.7-1.3.4-SNAPSHOT).
  Searched in the following locations:
      https://oss.sonatype.org/content/repositories/snapshots/org/bytedeco/javacpp-presets/fftw/3.3.7-1.3.4-SNAPSHOT/fftw-3.3.7-1.3.4-20171215.002004-377-linux-armhf.jar
> Could not find fftw-linux-ppc64le.jar (org.bytedeco.javacpp-presets:fftw:3.3.7-1.3.4-SNAPSHOT).
  Searched in the following locations:
      https://oss.sonatype.org/content/repositories/snapshots/org/bytedeco/javacpp-presets/fftw/3.3.7-1.3.4-SNAPSHOT/fftw-3.3.7-1.3.4-20171215.002004-377-linux-ppc64le.jar
> Could not find fftw-macosx-x86_64.jar (org.bytedeco.javacpp-presets:fftw:3.3.7-1.3.4-SNAPSHOT).
  Searched in the following locations:
      https://oss.sonatype.org/content/repositories/snapshots/org/bytedeco/javacpp-presets/fftw/3.3.7-1.3.4-SNAPSHOT/fftw-3.3.7-1.3.4-20171215.002004-377-macosx-x86_64.jar
> Could not find fftw-windows-x86.jar (org.bytedeco.javacpp-presets:fftw:3.3.7-1.3.4-SNAPSHOT).
  Searched in the following locations:
      https://oss.sonatype.org/content/repositories/snapshots/org/bytedeco/javacpp-presets/fftw/3.3.7-1.3.4-SNAPSHOT/fftw-3.3.7-1.3.4-20171215.002004-377-windows-x86.jar
```

**Comment by saudet** [comment]
Gradle doesn't work well with SNAPSHOT artifacts. Use Maven to download them to your `~/.m2/repository/` cache, and then Gradle will be able to use those.

**Comment by saudet** [comment]
Released with version 1.4!
http://search.maven.org/#search%7Cga%7C1%7Cfftw
Enjoy
